company-profiles/karat.md

```diff
@@ -0,0 +1,29 @@
+# Karat
+
+## Company blurb
+
+Karat created the world’s first Interviewing Cloud to solve the multi-decade global shortage of software engineers. The Interviewing Cloud is an always-on, scalable, and consistent human + technology solution for conducting predictive, fair, and enjoyable technical interviews 24/7. Karat’s global network of Interview Engineers, unparalleled market intelligence, and purpose-built interviewing infrastructure opens the pipeline for engineering talent, limits pedigree bias, and provides access for underrepresented candidates. The fastest growing and largest enterprises such as Roblox, American Express, Intuit, Compass, and Wayfair trust Karat to expand hiring capacity, unlock developer time, and raise engineering quality so they can innovate and grow.
+
+## Company size
+
+376 employees on LinkedIn
+
+## Remote status
+
+You will be working from home, any where in the world.
+
+## Region
+
+Worldwide
+
+## Company Specialties
```

Comment:
The build fails because you've changed the section title, please rename back to `Company technologies` please.